app/uk/gov/hmrc/soletraderidentificationfrontend/views/capture_overseas_tax_identifier_country_page.scala.html

```diff
@@ -0,0 +1,84 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components
+@import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
+@import uk.gov.hmrc.soletraderidentificationfrontend.models.PageConfig
+@import uk.gov.hmrc.soletraderidentificationfrontend.views.helpers.TitleHelper.title
+@import uk.gov.hmrc.soletraderidentificationfrontend.views.html.helpers._
+@import uk.gov.hmrc.soletraderidentificationfrontend.views.html.templates.govuk_template
+@import uk.gov.hmrc.soletraderidentificationfrontend.models.Country
+
+@this(layout: govuk_template,
+        h1: h1,
+        govukButton: GovukButton,
+        formWithCSRF: FormWithCSRF,
+        inputText: inputText,
+        errorSummary: errorSummary,
+        select: GovukSelect
+)
+
+@(journeyId: String,pageConfig: PageConfig, formAction: Call, form: Form[_], countries: Seq[Country])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+    @layout(pageConfig, title = title("tax-identifiers-country.title", form)) {
+
+    @formWithCSRF(action = formAction) {
+    @errorSummary(form.errors)
+
+    @h1(messages("tax-identifiers-country.heading"))
+
+    @select(components.Select(
+        id = "country",
+        name = "country",
+        label = Label(Some(messages("tax-identifiers-country.heading"))),
+        errorMessage = form("country").error.map(fe ⇒ ErrorMessage(content = HtmlContent(messages("error.tax_identifier_country")))),
+        items = Seq(SelectItem(value = Some(""), text = messages("country.select"))) ++ countries.map {
+            case Country(code, name) ⇒ SelectItem(
+                value = Some(code),
+                text = name,
+                selected = code == form("country").value.getOrElse(""),
+                attributes = Map("id" → s"country-$code")
+            )
+        },
+        formGroupClasses = "form-field-group"
+    ))
```

Comment:
I can still see the "missing form label" issue although you have label. Can you please check/
![Screenshot 2022-08-09 at 16 02 30](https://user-images.githubusercontent.com/75069573/183686204-b2bc7ca6-bde8-4007-8d0b-42d4d14de7fc.png)